testgrid/config.yaml

```diff
@@ -2720,6 +2720,16 @@ test_groups:
   alert_stale_results_hours: 24
   num_failures_to_alert: 1
 
+# Gardener Test Groups
+- name: ci-gardener-e2e-conformance-aws-v1.10
+  gcs_prefix: k8s-conformance-gardener/ci-gardener-e2e-conformance-aws-v1.10
+  alert_stale_results_hours: 24
+  num_failures_to_alert: 1
+- name: ci-gardener-e2e-conformance-gce-v1.10
+  gcs_prefix: k8s-conformance-gardener/ci-gardener-e2e-conformance-gce-v1.10
+  alert_stale_results_hours: 24
+  num_failures_to_alert: 1
```

Comment:
FYI this setting will be relevant if you [enable alert emails](https://github.com/kubernetes/test-infra/tree/master/testgrid#email-alerts)

Reply:
Thanks for the hint!